.github/workflows/update-deps.yaml

```
@@ -50,6 +61,16 @@ jobs:
             git diff && git status
             echo "HAS_GIT_DIFF=true" >> $GITHUB_OUTPUT
           fi
+      - name: Get Label
+        id: pr-label
+        if: ${{ steps.check_diff.outputs.HAS_GIT_DIFF == 'true' }}
+        run: |
```

actionlint

shellcheck reported issue in this script: SC1046:error:2:1: Couldn't find 'fi' for this 'if'

(shellcheck)